.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
       files: [
         '.eslintrc.js',
         'packages/kbn-pm/**/*',
+        'packages/kbn-es/**/*',
```

Comment:
In no way should this block this PR, but can we simplify this files array with a `packages/kbn-*/**/*`?

Reply:
Are we sure that all packages beginning with `kbn` will want the same rules applied?